### PR TITLE
User can create Grocery Ping

### PR DESCRIPTION
**Pull request by johanbounce**
### User Story
```
As a resident
In order to make a grocery run
I would like to be able to ping my community that I am going shopping
```
## Changes proposed in this pull request:
* Action-button to render form for creating a new trip (ping)
* Sending post request to API and receiving a response

## What I have learned working on this feature:
* REACT NATIVE 

## Packages/Gems added
* axios
* redux, react-redux 

## Screenshots (Client)
Skärmavbild 2020-04-13 kl  17 48 48

## Lo-Fi's (Client)
Pasted Image at 2020_04_11_01_36 pm

### CODE REVIEW

File: screens/NewTripForm.jsx [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+      {
+        ping: {
+          time: timevalue,
+          store: storevalue,
+        },
+      },
+      { headers: { "Content-Type": "application/json" } }
+    );
+    debugger;
+    dispatch({
```
Comment (emtalen) [inline screens/NewTripForm.jsx:41]: remove debugger

File: state/store/configureStore.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import { createStore } from 'redux'
+import rootReducer from '../reducers/rootReducer'
+
+const configureStore = () => {
+	return createStore(rootReducer)
+}
```
Comment (emtalen) [inline state/store/configureStore.js:6]: formatting on the return

**Review by emtalen — COMMENTED** [review]
We will need to do some refactoring in the newtrip component as a chore on a later stage, but I think this looks really good for a first featue in native!

File: state/store/configureStore.js (hunk 2 above)
Comment (johanbounce) [reply state/store/configureStore.js:6]: looked fine EMMA

File: screens/NewTripForm.jsx (hunk 1 above)
Comment (johanbounce) [reply screens/NewTripForm.jsx:41]: it's gåne

**Review by pgaunitz — APPROVED** [review]
LGDGTM

**Comment by kaylawoodbury** [comment]
> We will need to do some refactoring in the newtrip component as a chore on a later stage, but I think this looks really good for a first featue in native!

Yeah, I agree. But think that would be best as a chore once we have a better handle on Native :)

**Pull request merged**